openedx/openedx-platform
# fix: Dialog cutting off text

## Description

This PR changes the style of the "STAFF DEBUG INFO" dialog. Before in the New Experience view, when a user viewed the dialog and the text in a table exceed the width of the dialog, it would continue outside of the dialog and be hidden. Therefore, the user could not properly read all the debug information. Now when the text in the table exceeds the width of the dialog, it will automatically wrap within the table block. This change will impact the Course Author and people labeled as Staff.

**BEFORE**
![Animated GIF-downsized_large (1)](https://user-images.githubusercontent.com/42981026/121748229-9b2a8800-cad6-11eb-801b-1edcc679d859.gif)

**AFTER**
![Animated GIF-downsized_large](https://user-images.githubusercontent.com/42981026/121747995-40912c00-cad6-11eb-9736-9b6fea09b423.gif)

## Supporting information

Jira issue: [TNL-8332](https://openedx.atlassian.net/browse/TNL-8332)

## Testing instructions

- In the New Experience view, open a unit.
- Check that you are viewing the course as "Staff", if not switch to "Staff"
- Scroll to the bottom of the unit.
- Click on "STAFF DEBUG INFO"
- All text is visible and no information is cut off.

**NOTE:** Feel free to resize the window and see the text adjust accordingly.

## Deadline

None
